src/lib389/lib389/cli_idm/group.py

```diff
@@ -44,7 +44,7 @@ def get_dn(inst, basedn, log, args):
 
 
 def create(inst, basedn, log, args):
-    kwargs = _get_attributes(args, MUST_ATTRIBUTES)
+    kwargs = _get_attributes(args, MUST_ATTRIBUTES, MAY_ATTRIBUTES)
```

Comment:
I'm not sure why but when I run `dsidm inst group create` without any optional arguments, there's no interactive `Enter value for description:` prompt

Reply:
> I'm not sure why but when I run `dsidm inst group create` without any optional arguments, there's no interactive `Enter value for description:` prompt

Right, that because it's not required.  Otherwise it breaks CI tests when description is not set

Reply:
So you can also do this and it will interactivity still ask for requires attributes : 

    # dsidm inst group create --description="test"
    Enter value for cn:

Reply:
Okay, thinking more about that, that's good too.
I have no hard preference here:)

Reply:
> Okay, thinking more about that, that's good too. I have no hard preference here:)

Yeah it's an odd issue.  Would be nice to prompt for it, but it's not required, and everywhere else we only prompt for required attributes.